.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: CI
+
+on:
+  push:
```

Comment:
I just noticed this action runs on new PRs and changes to them. Let's change this to `workflow_dispatch` so we can run it manually instead of triggering it with unwanted actions.

Reply:
I don't understand your concern here. A good first review is whether the build passes or fails.

Reply:
Yes, but we don't want to push those builds to a docker registry on any pass when a PR is created or updated, unless we're going to remove the docker registry part.

Reply:
My bad. I just took another look and it seems the action doesn't push to a docker registry (is that correct?). If that's the case, let's ignore my comment.

Reply:
That's correct, this just builds the image locally, no push.